README.md

```diff
@@ -30,4 +29,32 @@ Finally, you'l need to run the bot, see the next section for instructions about
 
 ## Running the bot
 1. Open a terminal in the bot's folder
-2. Run `npm start`
\ No newline at end of file
+2. Run `npm start`
+
+If you want to add some flags, you can do so by doing `npm start --` and by adding a space before each flag. For example: `npm start -- -c` for the `-c` flag (see [the Execution flags section](#execution-flags) for a list of flags).
+
+## Execution flags
+| Flags             | Description                               |
+| ----------------- | ----------------------------------------- |
+| `-c`              | Adds colors to some console messages      |
+| `-d` or `--debug` | Enables debugging messages                |
+| `--no-backfill`   | Doesn't go back in time to parse messages |
```

Comment:
Might want to rewrite the description for this flag, to more clearly indicate what's happening.